batch-models/src/main/scala/org/ekstep/analytics/exhaust/OnDemandDruidExhaustJob.scala

```diff
@@ -187,7 +187,9 @@ object OnDemandDruidExhaustJob extends optional.Application with BaseReportsJob
             val storageConfig = getStorageConfig(config, AppConf.getConfig("collection.exhaust.store.prefix"))
             JobLogger.log("Total Requests are ", Some(Map("jobId" -> jobId, "totalRequests" -> requests.length)), INFO)
             val res = processRequest(request, reportConfig, storageConfig)
-            print(request, reportConfig, storageConfig)
+            JobLogger.log("Request is ",Some(request),INFO)
+            JobLogger.log("Report Config is ",Some(reportConfig),INFO)
+            JobLogger.log("Storage Config is ",Some(storageConfig),INFO)
```

Comment:
Please remove logging storage config as it might have azure key & secret info which cannot be exposed to logs.